cernopendata/modules/theme/assets/semantic-ui/scss/search.scss

```diff
@@ -4,6 +4,27 @@
   }
 }
 
+.ui.card {
+  .content {
+    padding: 10px 15px !important;
+
+    .header {
+      font-size: 0.9em !important;
```

Comment:
This value was set already on line 1 of this very same file.

Reply:
That value on line 1 controls the header font size of search results, while this line sets facets header font size. Definitely not ideal, but since we don't have any classes to attach styles to.. we have this mess

Reply:
I see, thanks for explaining, it tricked my mind...

> Definitely not ideal, but since we don't have any classes to attach styles to.. we have this mess

That's so true.. it's a bit of a mess not being able to add custom classes easily. I'll ask about this problem in Invenio chat to see if there's any other way than overriding the whole component.